nominal/core/connection.py

```diff
@@ -25,11 +30,25 @@ class Connection(HasRid):
     name: str
     description: str | None
     _tags: Mapping[str, Sequence[str]]
-    _clients: ClientsBunch = field(repr=False)
+    _clients: _Clients = field(repr=False)
     _nominal_data_source_rid: str | None = None
 
+    class _Clients(HasAuthHeader, Protocol):
```

Comment:
You can subclass from e.g. `Dataset._Clients` here so that you don't need to explicitly re-list all of the conjure clients for your dependencies. Check out how it's done for `Run._Clients` here: https://github.com/nominal-io/nominal-client/blob/3093c72bd78c9382cf6b94942e7afd50baa2ea2b/nominal/core/run.py#L34-L40